src/reactviews/pages/SchemaDesigner/schemaDesignerPage.tsx

```diff
@@ -8,14 +8,22 @@ import { SchemaDesignerContext } from "./schemaDesignerStateProvider";
 import "./schemaDesigner.css";
 import { SchemaDesignerToolbar } from "./toolbar/schemaDesignerToolbar";
 import { SchemaDesignerEditorDrawer } from "./editor/schemaDesignerEditorDrawer";
-import { SchemaDesignerCodeDrawer } from "./schemaDesignerCodeDrawer";
+import { SchemaDesignerDefinitionsPanel } from "./schemaDesignerDefinitionsPanel";
 import { SchemaDesignerFlow } from "./graph/SchemaDiagramFlow";
 import { SchemaDesignerFindTableWidget } from "./schemaDesignerFindTables";
-import { Spinner } from "@fluentui/react-components";
+import { makeStyles, Spinner } from "@fluentui/react-components";
 import { locConstants } from "../../common/locConstants";
+import { Panel, PanelGroup, PanelResizeHandle } from "react-resizable-panels";
```

Comment:
Introducing react-resizable-panels to reduce the code for making resizable panels.